### PR TITLE
feat(widget): obvious widget scrolling

### PR DESCRIPTION
**Pull request by RamilGaripov**
refs. metriport/metriport#378

### Description

- Obvious scrolling on the agreement and provider list. 
- Sticky "Continue" button with the text that explains what the user is agreeing to. 
- Sticky "Arrow Down" icon at the bottom of the provider list to indicate available scrolling. 

### Screenshots
https://imgur.com/a/ZU9SSdt

### CODE REVIEW

File: connect-widget/app/src/shared/components/WidgetContainer.tsx [hunk 1]
```diff
@@ -56,6 +57,14 @@ const WidgetContainer = ({ children }: WidgetContainerProps) => {
     },
   });
 
+  const [scrolled, setScrolled] = useState(false);
+
+  // eslint-disable-next-line @typescript-eslint/no-explicit-any
+  const handleScroll = (event: any) => {
+    if (event.currentTarget.scrollTop > 120) setScrolled(true);
```
Comment (Goncharo) [inline connect-widget/app/src/shared/components/WidgetContainer.tsx:64]: why are we hardcoding `120`? should we use a media query here?

File: connect-widget/app/src/shared/components/WidgetContainer.tsx [hunk 2]
```diff
@@ -93,9 +102,37 @@ const WidgetContainer = ({ children }: WidgetContainerProps) => {
...
+              position="sticky"
+              bottom={0}
+              left={0}
+              right={0}
+              display="flex"
+              justifyContent="center"
+            >
+              <ChevronDownIcon boxSize={12} />
+            </Box>
+          )}
```
Comment (Orta21) [inline connect-widget/app/src/shared/components/WidgetContainer.tsx:135]: I would suggest that we make the color of the arrow like a light gray or maybe even the faded metriport purple. Also could we have the icon be at the bottom but hover over the scroll?

**Review by Goncharo — APPROVED** [review]
Code LGTM - one thing to consider

File: connect-widget/app/src/shared/components/WidgetContainer.tsx (hunk 2 above)
Comment (RamilGaripov) [reply connect-widget/app/src/shared/components/WidgetContainer.tsx:135]: Should we maybe use the same color as the overall theme, but more faded instead?
Not sure what you mean by "hover over the scroll"

File: connect-widget/app/src/shared/components/WidgetContainer.tsx (hunk 1 above)
Comment (RamilGaripov) [reply connect-widget/app/src/shared/components/WidgetContainer.tsx:64]: I'll change it to a fraction of the window.innerHeight

**Review by Orta21 — APPROVED** [review]

File: connect-widget/app/src/shared/components/WidgetContainer.tsx [hunk 3]
```diff
@@ -56,6 +57,33 @@ const WidgetContainer = ({ children }: WidgetContainerProps) => {
     },
   });
 
+  const [displayIcon, setDisplayIcon] = useState(true);
+
+  useEffect(() => {
+    const handleResize = () => {
+      if (window.innerHeight < 1050) {
```
Comment (Goncharo) [inline connect-widget/app/src/shared/components/WidgetContainer.tsx:64]: is this the best way to do this?

whenever pixel are hardcoded it usually means that it won't work for all screen sizes

Comment (RamilGaripov) [reply connect-widget/app/src/shared/components/WidgetContainer.tsx]: Fixed

**Review by Goncharo — APPROVED** [review]
:shipit:

**Pull request merged**